src/components/Message.vue

```diff
@@ -231,34 +290,29 @@ export default defineComponent({
       await Http.setCookie(cookieOpts);
       const reqOpts = {
         method: 'PUT',
-        url: `https://scratch.mit.edu/site-api/users/followers/${o.actor_username}/add/?usernames=${this.session.username}`,
+        url: `https://scratch.mit.edu/site-api/users/curators-in/${o.gallery_id}/add/?usernames=${session.username}`,
         headers: {
           "x-requested-with": "XMLHttpRequest",
           "origin": "https://scratch.mit.edu/",
-          "referer": `https://scratch.mit.edu/`,
-          "x-token": this.session.token,
+          "referer": `https://scratch.mit.edu/studios/${o.gallery_id}/curators`,
+          "x-token": session.token,
           "x-csrftoken": "a",
           "cookie": "scratchcsrftoken=a;"
         }
       }
       const res = await Http.request(reqOpts);
       if (res.status == 200) {
-        this.toastNotif(`Followed user ${o.actor_username}`);
+        this.toastNotif(`You are a curator of ${o.title}`);
       } else {
         console.error(res.statusText);
       }
     },
-    async thankUser(o) {
```

Comment:
thank??

Reply:
Essentially this leads the current user to the following user's profile comments to thank them for the follow.

Reply:
I get it - but the function name is not correct here.

Reply:
What would you like me to rename it?